topics/assembly/tutorials/general-introduction/tours/tour.yaml

```diff
@@ -0,0 +1,281 @@
+        <li>K-mer: 29</li>
+        <li>Input file type: Fastq</li>
+        <li>Single or paired end reads: Paired</li>
+        <li>Select first set of reads: mutant_R1.fastq</li>
+        <li>Select second set of reads: mutant_R2.fastq</li>
+      </ul>
+    position: left
+  - title: Step 15. Assemble reads with Velvet
+    element: '.history-right-panel .list-items > *:first'
+    content: Now inspect each of the outputs.
+    position: left
+  - title: Assemble reads with Velvet. The contigs file
+    content: >-
+      The Contigs file will show each contig with the k-mer length and k-mer
+      coverage listed as part of the header (however, these are just called
+      length and coverage).<ul>
+        <li>K-mer length: For the value of k chosen in the assembly, a measure of how many k-mers overlap (by 1 bp each overlap) to give this length.</li>
+        <li>K-mer coverage: For the value of k chosen in the assembly, a measure of how many k-mers overlap each base position (in the assembly).</li>
+    backdrop: false
+  - title: Assemble reads with Velvet. The Contigs stats file
+    content: >-
+      The Contigs stats file will show a list of these k-mer lengths and k-mer
+      coverages.
+    backdrop: false
+  - title: Collect some statistics on the contigs
+    content: We will now collect some basic statistics on our assembly.
+    backdrop: true
+  - title: Step 16. Collect fasta statistics on the contigs
+    element: '#tool-search-query'
+    content: Search for fasta statistics tool
+    placement: right
+    textinsert: fasta statistics
+  - title: Step 17. Collect fasta statistics on the contigs
+    element: '#tool-search'
+    content: Click on the "fasta statistics" tool to open it
+    placement: right
+    postclick:
+      - >-
+        a[href$="/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fsimon-gladman%2Ffasta_stats%2Ffasta-stats%2F1.0.0"]
+  - title: Step 18. Collect fasta statistics on the contigs
+    element: '#center-panel'
+    content: Execute the tool on the input data with default settings.
+    position: left
+  - title: Step 19. Collect fasta statistics on the contigs
+    element: '.history-right-panel .list-items > *:first'
+    content: |-
+      Inspect the output<ul>
+        <li>Click on the eye ("View data") on the right of the file name in the history</li>
+        <li>Inspect the content of the file on the middle</li>
+      </ul>
+    position: left
+  - title: Collect fasta statistics on the contigs
+    content: |-
+      Inspect the output<ul>
+        <li>num_seq: the number of contigs in the FASTA file</li>
+        <li>num_bp: the number of assembled bases. Roughly proportional to genome size.</li>
+        <li>len_max: the biggest contig.</li>
+        <li>len_N50: N50 is a contig size. If contigs were ordered from small to large, half of all the nucleotides will be in contigs this size or larger.</li>
+      </ul>
+    backdrop: true
+  - title: Discussion
```

Comment:
Are the charts spoken about here displayed anywhere? Otherwise it will be difficult for someone to answer the questions posed in the next box.

Reply:
yep, they are in a corresponding tutorial